rockcraft.yaml

```diff
@@ -0,0 +1,27 @@
+name: gunicorn-k8s-operator
+summary: An OCI image for use with the gunicorn-k8s-operator.
+description: Runs a basic gunicorn workload that dumps environment variables to a web page.
+license: Apache-2.0
+
+version: "1.0"
+base: bare
+build_base: ubuntu:22.04
+cmd: [/srv/gunicorn/run]
+env:
+  - APP_WSGI: 'app:app'
+  - APP_NAME: 'my-awesome-app'
```

Comment:
Is this supposed to change or is it an example?

Reply:
it could be anything, but this is what we currently use in the dockerfile as well. The image we're building is a simple image that dumps environment variables to a plain text web page, so it's just a demo really, more than anything. The intention is that people would replace the image at deploy time with the one they want to use.